sbatch_scripts/sample_generate_sbatch.sh

```
@@ -1,20 +1,24 @@
+#SBATCH --array=1-10
 
 module load python
 module load gcc
 
-python -m uv run generate --config-path /path/to/config_data_generation.yaml --log-level WARNING --output my_generated_data
+# MLflow environment variables
+export MLFLOW_EXPERIMENT_NAME=ddm-data-generation
+export MLFLOW_TRACKING_URI=sqlite:///mlflow.db
+
+python -m pip install --user uv
+python -m uv run generate --config-path configs/examples/data_generation.yaml --log-level WARNING --output /path/to/output --mlflow-run-name ddm-worker-$SLURM_ARRAY_TASK_ID --mlflow-experiment-name ddm-data-generation
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)